### DISCUSSION TITLE
TypeError: (0 , react__WEBPACK_IMPORTED_MODULE_5__.createContext) is not a function when using with NextJs

### DISCUSSION THREAD

**Discussion by praveen7557** [JedWatson/react-select#5239] [Q&A]
When trying use react-select in NextJs TypeScript getting `TypeError: (0 , react__WEBPACK_IMPORTED_MODULE_5__.createContext) is not a function` error.

Dependencies

<img width="372" alt="Screenshot 2022-06-22 at 4 29 09 PM" src="https://user-images.githubusercontent.com/13882201/175013128-dd82d4dd-c743-4d08-8f65-0cda37fb3938.png">

Seems to be working in codesandbox but not in my local.

**Comment by manjushsh** [comment]
@praveen7557  May be due to versions.
If it is working in sandbox, try deleting node modules and package-lock and run npm install again
[Stack Overflow](https://stackoverflow.com/a/54521764)

**Reply by praveen7557** [reply]
@manjushsh even I was facing no issues while using codesandbox. I tried deleting node_modules and yarn.lock, still not working.
